### PR TITLE
Add list structure

### PR DESCRIPTION
**Pull request by Yazino12**
### In this pull:

- Setup project with webpack
- Modify starting files and structure, styling and script
- Create an array of objects to store data for list
- Add function to populate data from array to corresponding HTML placeholder element

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 > This simple web page is built using webpack and served by a webpack dev server
 
+<h2 align="center">
+  Demo 📝
+</h2>
+
+<img src="https://github.com/Yazino12/todo-list/blob/dev/src/assets/demo.png?raw=true" width="1433" height="600">
+
```
Comment (mikethreels) [inline README.md:12]: - [x] please make sure to use[ the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Yasin, 

Good job on the project, the code is very clean and it works exactly as is expected 👍 
However, there is one issue that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Yazino12) [reply README.md:12]: Thanks for pointing that out @mikethreels, will implement these changes right away 😊

Comment (Yazino12) [reply README.md:12]: Just one thing to point out is, if I did not add a license at the beginning of creating the repository is it possible to add a licence later, and if yes would that be valid? @mikethreels

Comment (mikethreels) [reply README.md:12]: Yes, that is no problem, just add the MIT.md file which is included in [the template repository](https://github.com/microverseinc/readme-template) and adjust it a little so it relates to your project.

**Review by t-yanick — APPROVED** [review]
Hi @Yazino12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**